packages/card/docs.mdx

```diff
@@ -0,0 +1,205 @@
+---
+componentName: 'Card'
+---
+
+Cards are used to promote other marketing pages.
```

Comment:
turbo nit: this feels like one possible use for cards, but this description could be a bit more nuanced.

Reply:
Agreed! This description is mainly just a rewording of the one by design for now.  It's worth workshopping a more holistic description here :+1:

Reply:
@mwickett Updated the description here:
https://github.com/hashicorp/react-components/pull/694/commits/5d5a9cb556ef5b2334c62622cfaaf1f53355469d

It's more detailed but I still think it's missing something 🤔 Thoughts?